starknet_py/contract.py

```diff
@@ -248,24 +264,24 @@ async def invoke(
         """
         Invokes a method.
 
-        :param max_fee: Max amount of Wei to be paid when executing transaction
-        :param auto_estimate: Use automatic fee estimation, not recommend as it may lead to high costs
-        :return: InvokeResult
+        :param max_fee: Max amount of Wei to be paid when executing transaction.
+        :param auto_estimate: Use automatic fee estimation, not recommend as it may lead to high costs.
+        :return: InvokeResult.
         """
         if max_fee is not None:
             self.max_fee = max_fee
 
-        transaction = await self._account_client.sign_invoke_transaction(
+        transaction = await self._account.sign_invoke_transaction(
             calls=self,
             max_fee=self.max_fee,
             auto_estimate=auto_estimate,
-            version=self.version,
         )
-        response = await self._account_client.send_transaction(transaction)
+
+        response = await self._client.send_transaction(transaction)
 
         invoke_result = InvokeResult(
             hash=response.transaction_hash,  # noinspection PyTypeChecker
-            _client=self._account_client,
+            _client=self._client,
             contract=self._contract_data,
             invoke_transaction=transaction,
         )
```

Comment:
InvokeResult will now contain a `client` i.e. Gateway or FullNode even if `AccountClient` was used to create a Contract instance. Should this be included in the migration guide?

Reply:
It's private property so I don't think so

Reply:
It is a private property, right? If anyone is using it they are to blame.

Reply:
Agreed, not adding it then.